app/forms/fields/date_field.py

```diff
@@ -30,11 +32,28 @@ def data(self):
 
 
 class DateField(FormField):
-    def __init__(self, validators, **kwargs):
+    def __init__(
+        self,
+        *,
+        validators: Sequence[Callable],
```

Comment:
Instead of using generic Callable, use our types where possible. There is validator types variables in the handlers files, for example here it could be `DateValidatorTypes` from `date_handlers.py`. You will most likely need to move them to `types.py` to avoid circular imports.

Applies to other fields also.

Reply:
Ah nice that makes sense, thanks 👍